src/registrar/templates/admin/app_list.html

```diff
@@ -4,12 +4,20 @@
   {% for app in app_list %}
     <div class="app-{{ app.app_label }} module{% if app.app_url in request.path|urlencode %} current-app{% endif %}">
       <table>
-        <caption>
-          <a href="{{ app.app_url }}" class="section" title="{% blocktranslate with name=app.name %}Models in the {{ name }} application{% endblocktranslate %}">{{ app.name }}</a>
```

Comment:
Do we need to keep this <a>? Its doing a redirect which I don't believe we are doing in the new approach

Reply:
If you remember from the UX PR @zandercymatics, we're removed the link in any case. I don't see it used anywhere else on this template.